catalog/core/catalog-core-directorymonitor/src/main/java/org/codice/ddf/catalog/content/monitor/ContentDirectoryMonitor.java

```diff
@@ -151,143 +162,134 @@ public void setAttributeOverrides(List<String> attributeOverrides) {
-            // administrator or this ContentDirectoryMonitor is deleted.
-            this.routeCollection = routeBuilder.getRouteCollection()
-                    .getRoutes();
-
-            // Start route that was just added.
-            // If the route was just added for the first time, i.e., this not a bundle
-            // restart, then this method will do nothing since the addRoutes() above
-            // already started the route. But for bundle (or system) restart this call
-            // is needed since the addRoutes() for whatever reason did not start the route.
-            startRoutes();
-
+            setRouteCollection(routeBuilder);
+        } catch (InterruptedException e) {
+            Thread.currentThread()
+                    .interrupt();
+            LOGGER.warn("CDM interrupted while waiting for Camel component 'content'. ");
+        } catch (Exception e) {
+            LOGGER.warn(
+                    "Unable to configure Camel route - this Content Directory Monitor will be unusable");
+        } finally {
             if (LOGGER.isDebugEnabled()) {
                 dumpCamelContext("after configureCamelRoute()");
             }
-        } catch (Exception e) {
-            LOGGER.warn(
-                    "Unable to configure Camel route - this Content Directory Monitor will be unusable",
-                    e);
         }
-
-        LOGGER.trace("EXITING: configureCamelRoute");
-    }
-
-    public List<RouteDefinition> getRouteDefinitions() {
-        return camelContext.getRouteDefinitions();
     }
 
-    private void startRoutes() {
-        LOGGER.trace("ENTERING: startRoutes");
-        List<RouteDefinition> routeDefinitions = camelContext.getRouteDefinitions();
-        for (RouteDefinition routeDef : routeDefinitions) {
-            startRoute(routeDef);
-        }
-        LOGGER.trace("EXITING: startRoutes");
+    /*
+        Assign this CDM's routeCollection to the routes generated by this CDM's RouteBuilder.
+     */
+    private void setRouteCollection(RouteBuilder routeBuilder) {
+        routeCollection = routeBuilder.getRouteCollection()
+                .getRoutes();
     }
 
-    private void startRoute(RouteDefinition routeDef) {
-        String routeId = routeDef.getId();
-        try {
-            if (isMyRoute(routeId)) {
-                ServiceStatus routeStatus = camelContext.getRouteStatus(routeId);
-                // Only start the route if it is not already started
-                if (routeStatus == null || !routeStatus.isStarted()) {
-                    LOGGER.trace("Starting route with ID = {}", routeId);
-                    camelContext.startRoute(routeDef);  //DEPRECATED
-                    // this method does not reliably start a route that was created, then
-                    // app shutdown, and restarted
-                    //                camelContext.startRoute(routeId);
-                }
```

Comment:
Method removed. Completely unnecessary since `camelContext.addRoutes(routeBuilder)` does the starting of the routes for us.